.github/workflows/increment-version.yml

```diff
@@ -2,7 +2,7 @@ name: Increment version when languages are updated
 
 on:
   push:
-    branches: [ master ]
+    branches: [ Matrix, Nexus, Omega ]
```

Comment:
I will change that, because it breaks the my flow on releasing branches: only master for here.